### ISSUE TITLE
Acutually the PyTorch version needed to run your codes is higher than 1.2.0

### ISSUE THREAD

**Issue by yyl-github-1896** [lafeat/lafeat#1]
I am very interested in your paper, so I want to reproduce your experiments yesterday. My Python version is 3.7.9 and my PyTorch version is exactly 1.2.0   When I ran 
python3 attack.py \
    --verbose --batch-size=${your_batch_size:-2000} \
    --multi-targeted --num-iterations=1000 \
    --logits-model=models/trades_new.pt  # your trained logits
It reported a bug in /lafeat/lafeat/attack.py  line 113: logits_prev = scale_17 * out_adv_256_17 * mask_256_17 + (1 - scale_17 * mask_256_17) * out_adv, saying that I should have used torch.float to do this calculation other than Char. 
It turned out that the variable "mask_256_17" is in type "torch.int8", which was interpreted as Char in PyTorch 1.2.0
To fix this bug, we have two choices: (1) Change the return type of the function "check_right_index(self, output, labels)". We can modify /lafeat/lafeat/attack.py  line 42: from "mask = output_index.to(dtype=torch.int8)" to "mask = output_index.to(dtype=torch.float)".  (2) Upgrade our PyTorch version. I found that at least in PyTorch 1.5.1, the type torch.in8 can be automatically converted to torch.float when it is going to be operated with torch.float
